src/actions/accountActions.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import axios from "axios";
+
+class AccountAction extends React.Component {
```

Comment:
This does not need to be a component. It can be an object with functions within it.